nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -202,22 +202,22 @@ in
 
       workstation = lib.mkOption {
         type = lib.types.bool;
-        default = false;
+        default = true;
         description = ''
           Whether to register a service of type "_workstation._tcp" on the local LAN.
         '';
```

Comment:
same here https://github.com/avahi/avahi/blob/1dade81cbf4d3e3f49784e44e543cd77046c79df/man/avahi-daemon.conf.5.xml.in#L263-L269